README.md

```diff
@@ -4,128 +4,28 @@ Genny 🧞‍
-    --build-from-source \
-    --include-test      \
-    --with-icu4c
-```
-
-Other errors, run `brew doctor`.
-
-#### Notes for macOS Mojave
-
-Mojave doesn't have `/usr/local` on system roots, so you need to set
-environment variables for clang/ldd to find libraries provided by
-homebrew.
-
-Put the following in your shell profile (`~/.bashrc` or `~/.zshrc` etc):
-
-```sh
-export CPLUS_INCLUDE_PATH="$(brew --prefix)/include"
-export LIBRARY_PATH="$(brew --prefix)/lib/:$LIBRARY_PATH"
-```
-
-This needs to be run before you run `cmake`. If you have already run
-`cmake`, then `rm -rf build/*` after putting this in your profile
-and restarting your shell.
-
-TODO: TIG-1263 This is kind of a hack; using built-in package-location
-mechanisms would avoid having to have OS-specific hacks like this.
-
-### Linux Distributions
-
-Have recent versions of non-vendored dependent packages in your system,
-using the package manger. Generally this is:
-
-- boost
-- cmake (>=3.10)
-- grpc (>=1.16)
-- icu
-- mongo-cxx-driver
-- protobuf (3.6.1)
-
-To build genny use the following commands:
+Ubuntu 16.04+: sudo apt install build-essential
+Red Hat/CentOS 7+: sudo yum groupinstall "Development Tools"
+Arch: Grab a beer. Everything should already be set up.
+macOS: xcode-select --install
+Windows: https://visualstudio.microsoft.com/
 
-```sh
-cmake -E chdir "build" cmake ..
-make -j8 -C "build" genny
-```
-
-You only need to run cmake once. Other useful targets include:
-
-- all
-- gennylib_test driver_test (builds tests)
-- test (run's tests if they're built)
-
-#### Ubuntu 18.04 LTS
-
-For C++17 support you need at least Ubuntu 18.04. (Before you say
-"mongodbtoolchain", note that it doesn't provide cmake.)
-
-```sh
-apt install -y \
-    build-essential \
-    cmake \
-    software-properties-common \
-    clang-6.0 \ # optional
-    libboost-all-dev
-```
-You also need `libgrpc++-dev` and `libprotobuf-dev`, but here genny is more picky about the
-versions. They need to be >=1.16 and exactly 3.6.1 respectively. To install those from source:
-
-- protobuf: https://github.com/protocolbuffers/protobuf/blob/master/src/README.md
-- grpc: https://github.com/grpc/grpc/blob/master/src/cpp/README.md
+1. Make sure you have a C++17 compatible compiler and Python 3. 
+The ones from mongodbtoolchain are safe bets if you're unsure.
+(mongodbtoolchain is internal to MongoDB).
 
-If you already installed the wrong versions with apt, you are better off
-uninstalling them:
-
-```sh
-apt remove libgrpc++-dev libprotobuf-dev
-```
-
-Finally, install mongo C++ driver from source too:
-
-- https://mongodb.github.io/mongo-cxx-driver/mongocxx-v3/installation/
-- Note that you need to specifically tell it to install into `/usr/local`!
-
-Note that on Ubuntu 18.04 you need a custom linker option:
-
-```sh
-cmake -E chdir "build" cmake -DCMAKE_EXE_LINKER_FLAGS=-Wl,--no-as-needed ..
-make -j8 -C "build" genny
-```
+1. `python3 lamp.py [--linux-distro ubuntu1804/rhel7/amazon2/arch]`
```

Comment:
I ran this locally and while it works the warnings it produces may scare a new user to think that things aren't right. Mention that 

1. `#  define MONGOCXX_DEPRECATED __attribute__ ((__deprecated__))` errors are known (and link to ticket to update to latest driver to kill the warning0
2. `boost::system::error_category::std_category::equivalent` errors are known (I think..we didn't see those prior to the toolchain so is there something still not quite right there?)

Reply:
> If it can't be fixed easily (because it's boost), would you be OK with a banner + followup ticket for now?

Ya fine by me but compiling warning-free is definitely something I value highly so I'd want to prioritize as a priority bug-fix. Just my $0.02.